app/views/documents/form_partials/_tags.html.erb

```diff
@@ -1,6 +1,6 @@
 <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
-<div class="govuk-form-group <%= form.object.errors[:tags].any? ? 'govuk-form-group--error' : '' %>" data-controller="show-hide">
+<div class="govuk-form-group <%= form.object.errors[:tags].any? ? 'govuk-form-group--error' : '' %>">
```

Comment:
Is this needed since we're now using `govuk_check_boxes_fieldset` inside the tab which will add the error styling